Look for make-cert.cnf in $DPATH, and exit if an error occurred.

https://github.com/DALnet/bahamut/issues/232
--- tools/make-cert.tmpl.orig	2022-08-16 10:40:51.000000000 -0500
+++ tools/make-cert.tmpl	2023-07-06 21:31:15.000000000 -0500
@@ -26,6 +26,6 @@
         exit 1
     fi
 fi
-openssl req -new -x509 -days 1825 -nodes -config make-cert.cnf -out "$DPATH/ircd.crt" -keyout "$DPATH/ircd.key" $RANDCMD
+openssl req -new -x509 -days 1825 -nodes -config "$DPATH/make-cert.cnf" -out "$DPATH/ircd.crt" -keyout "$DPATH/ircd.key" $RANDCMD || exit $?
 openssl x509 -subject -dates -fingerprint -noout -in "$DPATH/ircd.crt"
 
